test/unit/jasmine-boot.js

```diff
@@ -60,6 +60,7 @@ async function initializePDFJS(callback) {
       "pdfjs-test/unit/core_utils_spec.js",
       "pdfjs-test/unit/crypto_spec.js",
       "pdfjs-test/unit/custom_spec.js",
+      "pdfjs-test/unit/default_appearance_spec.js",
```

Comment:
Please add this to https://github.com/mozilla/pdf.js/blob/master/test/unit/clitests.json as well.